src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -22,7 +22,6 @@
     <Compile Include="System\Net\IWebRequestCreate.cs" />
     <Compile Include="System\Net\ProtocolViolationException.cs" />
     <Compile Include="System\Net\RequestStream.cs" />
-    <Compile Include="System\Net\SystemWebProxy.cs" />
```

Comment:
Removing this class will break this code in WinHttpHandler:

https://github.com/dotnet/corefx/blob/fe8db47cb290eff654e8419ce6d615e9bb2ce189/src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs#L709-L714

We can choose to do that. But we should adjust the related WinHttpHandler code (perhaps use HttpClient.DefaultProxy now).

Reply:
I assumed UseWinInetProxy had the same behavior as HttpClient.DefaultProxy; does it not?  If it does, then it wouldn't actually break it... the proxy will simply work and we'll never end up in the catch block.

Reply:
We also might determine that once #41182 is implemented, then we don't need to support this behavior since callers would be using WinHttpHandler explicitly and could set `UseWinInetProxy` directly.

Reply:
>I assumed UseWinInetProxy had the same behavior as HttpClient.DefaultProxy (when no environment variables were defined, as they'd take precedence); does it not?

Yes, but only in the initial default case of using HttpClient.DefaultProxy. That property also has a setter. So, someone could set it to any custom IWebProxy.

Reply:
I'm trying to think through the various combinations here. Perhaps everything will be ok after removing this code. But we might need to revisit this change afterwards.

Reply:
> So, someone could set it to any custom IWebProxy.

Sure, but that's no different from today.

So I'm not seeing how anything is negatively affected.

Reply:
Correct. I'm thinking in real-time with these comments we're discussing. So, I won't block on this right now. It's just something that occurred to me after looking at this PR.

Reply:
> Correct. I'm thinking in real-time with these comments we're discussing

Heh, ok. :smile: